scripts/simex

```diff
@@ -253,6 +253,37 @@ builds_make_parser.set_defaults(cmd=do_builds_make)
 
 # ---------------------------------------------------------------------------------------
 
+def do_dev_builds(args):
```

Comment:
Rename to `do_develop` (for consistency with other commands).

Reply:
done